app/services/onestop_id.rb

```diff
@@ -56,68 +116,141 @@ def self.create_identifier(feed_onestop_id, entity_prefix, entity_id)
     ).to_s
   end
 
-  def self.validate_onestop_id_string(onestop_id, expected_entity_type: nil)
-    errors = []
-    is_a_valid_onestop_id = true
+  def OnestopId.validate_onestop_id_string(onestop_id)
+    binding.pry
+    klass = lookup(onestop_id)
+    return false, ['invalid prefix'] unless klass
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.